### PR TITLE
Status string updates

### PR DESCRIPTION
**Pull request by Randy424**

### CODE REVIEW

File: frontend/src/lib/get-cluster.ts [hunk 1]
```diff
@@ -38,6 +38,7 @@ export enum ClusterStatus {
     'posthookjob' = 'posthookjob',
     'posthookfailed' = 'posthookfailed',
     'importfailed' = 'importfailed',
+    'nojobs' = 'nojobs',
```
Comment (showeimer) [inline frontend/src/lib/get-cluster.ts:41]: This doesn't look needed when I look at the changes

File: frontend/public/locales/en/cluster.json [hunk 2]
```diff
@@ -345,15 +346,15 @@
...
+    "status.prehookfailed.message": "One or more of the cluster prehook jobs failed.",
+    "status.posthookjob.message": "The cluster posthook jobs are currently in progress.",
+    "status.posthookfailed.message": "One or more of the cluster posthook jobs failed.",
     "status.importfailed.message": "The cluster failed to attach to the hub cluster.",
     "status.unknown.alert.title": "The cluster is not reachable",
     "status.offline.alert.title": "The cluster is not available",
-    "status.prehookfailed.alert.title": "The cluster pre-creation hooks failed",
-    "status.posthookfailed.alert.title": "The cluster post-creation hooks failed",
+    "status.prehookfailed.alert.title": "The cluster prehook failed",
+    "status.posthookfailed.alert.title": "The cluster posthook failed",
```
Comment (showeimer) [inline frontend/public/locales/en/cluster.json:357]: since above it's `prehook jobs`, should this be `The cluster prehook jobs failed` ?

**Review by showeimer — CHANGES_REQUESTED** [review]

File: frontend/src/lib/get-cluster.ts (hunk 1 above)
Comment (Randy424) [reply frontend/src/lib/get-cluster.ts:41]: Brain malfunctioned here, thanks.

**Review by jamestalton — APPROVED** [review]
lgtm

**Review by showeimer — APPROVED** [review]

**Pull request merged**